packages/binding-mbus/package.json

```diff
@@ -35,5 +35,12 @@
   "scripts": {
     "build": "tsc",
     "test": "mocha --require ts-node/register test/*"
+  },
+  "bugs": {
+    "url": "https://github.com/thingweb/node-wot/issues"
+  },
+  "homepage": "https://github.com/thingweb/node-wot/tree/master/packages/binding-mbus#readme",
```

Comment:
Link is broken,  should be
https://github.com/eclipse/thingweb.node-wot/tree/master/packages/binding-mbus#readme